data/integrations.yml

```diff
@@ -6,6 +6,7 @@ slack:
     - run_error
     - webhook_timeout
     - run_test_failure
+    - integration_test
```

Comment:
Instead of adding this to every integration, it would be better to have a method in the base class along the lines of:
```
event_supported?(event_type)
event_type == "test_integrations" ||  SUPPORTED_EVENTS.include?(event_type)
end
```